File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq_dict = {}
        for num in nums:
            freq_dict[num] = freq_dict.get(num, 0) + 1
            sorted_items = sorted(freq_dict.items(), key=lambda item: item[1], reverse=True)
        result = []
        for i in range(k):
            result.append(sorted_items[i][0])
        return result

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.go
func findMaxConsecutiveOnes(nums []int) int {
	var count, max int

	for i := range nums {
		if nums[i] == 1 {
			count++
			if count > max{
				max = count
			}
		} else {
			if count > max{
				max = count
			}
			count = 0
		}
	}
	return max
}
